motoman_sda10f_moveit_config/launch/default_warehouse_db.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
 
-  <arg name="reset" default="false"/>
+  <arg name="reset" default="false" doc="Reset DB"/>
```

Comment:
This file (and all others in MoveIt) configurations will all be overwritten if the Setup Assistant is ever run again.

Do you feel it's worth it to manually add doc strings here, instead of updating the MSA templates upstream?